bap-vibes/src/arm_selector.ml

```diff
@@ -1267,7 +1305,35 @@ let is_nop (op : Ir.operation) : bool =
 let filter_nops (ops : Ir.operation list) : Ir.operation list =
   List.filter ops ~f:(fun o -> not (is_nop o))
 
-let peephole (ir : Ir.t) : Ir.t =
+let implicit_fallthroughs (ir : Ir.t) : Ir.t =
```

Comment:
This is run on fully instantiated vibes ir? Maybe there is some way to move this over to Ir.ml parametrized over what is considered a fallthrough instruction.

Reply:
I think this can be left for a future PR, and would be motivated better when we plan to target more instruction sets.